pkg/controller/localvolumediscovery/localvolumediscovery_controller.go

```diff
@@ -36,6 +36,7 @@ var (
 
 const (
 	DiskMakerDiscovery = "diskmaker-discovery"
+	PriorityClassName  = "openshift-user-critical"
```

Comment:
This won't work as a global constant (being imported by the `nodedaemon/daemonsets.go`, as there's a circular dependency.

I'd recommend moving this to `pkg/common/daemonsets_utils.go`, which is already imported in both locations.

Reply:
👍🏽  🙏🏽